### DISCUSSION TITLE
[BUG] Creating a user with the .js extension causes errors in the loading of the profile page's CSS.

### DISCUSSION THREAD

**Discussion by anabergerr** [codewars/codewars.com#3091] [General]
When creating my user using .js at the end, I noticed that the CSS of the page doesn't load. This only happens when I use .js.

Here's a screenshot of the page and the link.

https://www.codewars.com/users/ana.js

![image](https://github.com/codewars/codewars.com/assets/89489383/ca92f62e-98d8-4fae-9a82-ac9e44daed47)

**Comment by devcarolzita** [comment]
Oh 😮  nice error! 

What happens behind the Code Wars servers?

**Comment by hobovsky** [comment]
I created a [bug report](https://github.com/codewars/codewars.com/issues/3092).
